### PR TITLE
854 miovision `open_issues` for tracking and communication

### PR DESCRIPTION
**Pull request by gabrielwol**
## What this pull request accomplishes:
- adds `miovision_api.open_issues`, a view which references `anomalous_ranges` in a format that will be easier to share with outside parties (adds intersection / classification names+ids common to Miovision) and `last_week_volume` to find the volume (if any) from the last 7 days. 
- adds a new task to `miovision_check` DAG that runs weekly on Friday (execution date of Thursday) to report on any open ended issues with volumes so that they don't get stale.

## Issue(s) this solves:
- #854

## What, in particular, needs to reviewed:
- 

## What needs to be done by a sysadmin after this PR is merged
- [x] create view in miovision_api (+ delete from gwolofs + edit DAG code to pull from correct schema)
- [x] test DAG: https://bditto.slack.com/archives/C04BWRWVB3L/p1712696273255669
- [x] add `api_name` to view in place of `intersection_name` once #871 gets merged

### CODE REVIEW

File: dags/miovision_check.py [hunk 1]
```diff
@@ -95,6 +98,40 @@ def miovision_check_dag():
...
+    check_open_anomalous_ranges = SQLCheckOperatorWithReturnValue(
+        task_id="check_open_anomalous_ranges",
+        sql="""WITH ars AS (
+            SELECT
+                uid,
+                intersection_name,
+                classification_uid,
+                notes
+            FROM gwolofs.open_issues
+            WHERE last_week_volume > 0
```
[the remark this answers is not in the document]
Comment (chmnata) [reply dags/miovision_check.py:115]: cause we already have volume is null in the daily alert?

File: dags/miovision_check.py [hunk 2]
```diff
@@ -95,6 +98,40 @@ def miovision_check_dag():
...
+            ORDER BY uid
+        )
+
+        SELECT
+            NOT(COUNT(*) > 0) AS _check,
+            CASE WHEN COUNT(*) = 1 THEN 'There is ' ELSE 'There are ' END || COUNT(*) ||
+                ' open ended anomalous_range(s) with non-zero volumes in the last 7 days: (uid, intersection_name, classification_uid, notes)'
```
Comment (chmnata) [inline dags/miovision_check.py:122]: I think we can get rid of the stuff in the brackets

File: dags/miovision_check.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (chmnata) [reply dags/miovision_check.py:115]: We have 33 ongoing volume is null in the open_issue view 🤔

**Review by chmnata — CHANGES_REQUESTED** [review]

File: dags/miovision_check.py [hunk 3]
```diff
@@ -95,6 +98,40 @@ def miovision_check_dag():
...
+        start_date = datetime.strptime(ds, '%Y-%m-%d')
+        return start_date.isoweekday() == 4
+    
+    check_open_anomalous_ranges = SQLCheckOperatorWithReturnValue(
+        task_id="check_open_anomalous_ranges",
+        sql="""WITH ars AS (
+            SELECT
+                uid,
+                intersection_name,
+                classification_uid,
```
[the remark this answers is not in the document]
Comment (gabrielwol) [reply dags/miovision_check.py]: Added both! (need to be able to tell the difference between bikes approach and tmc)

File: dags/miovision_check.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (gabrielwol) [reply dags/miovision_check.py:116]: I'm thinking that this alert will be a nudge to close open alerts that do have volumes, especially open ended ones. All the 33 (a lot of those are just no bike approaches) would be overwhelming I think.

File: dags/miovision_check.py (hunk 2 above)
Comment (gabrielwol) [reply dags/miovision_check.py]: Gonna leave, mostly to identify the cryptic uid (anomalous_range.uid) field.

**Comment by gabrielwol** [comment]
See the latest draft message here: https://bditto.slack.com/archives/C04BWRWVB3L/p1713557743044479

File: dags/miovision_check.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (chmnata) [reply dags/miovision_check.py]: Do we regularly look at the no volumes one for report back? Wanted to make sure we are not forgetting these

**Comment by chmnata** [comment]
Agree that the alert message can be more readable, what about we just print the intersection name + volume ? 

There are 12 open ended anomalous_range(s) with non-zero volumes in the last 7 days:
uid: 801, Adelaide Street West and Bay Street (3), volume: 31356
uid: 852, College Street and Bathurst Street (69), volume: 14782
uid: 854, King Street East and Lower River Street (71), volume: 4311
uid: 855, Queen Street East and River Street (72), volume: 5886

File: dags/miovision_check.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (gabrielwol) [reply dags/miovision_check.py]: There are a lot of those too - I think we'll need a regular, Chris-assisted process to nudge Miovision about them.
My thinking was that the ones with non-zero volume are in our court, while the ones with zero volume are in their court.

**Comment by gabrielwol** [comment]
@chmnata Ready for re-review.
I got the slack message formatting to be more concise by changing it from an attachment to a quoted block: https://bditto.slack.com/archives/C04BWRWVB3L/p1714076754656569

**Review by chmnata — APPROVED** [review]

**Comment by chmnata** [comment]
Thanks! the message formatting is much better meow

**Pull request merged**